### PR TITLE
update magento README

### PR DESCRIPTION
**Pull request by bryanlarsen**
- add minikube installation instructions
- add metal installation instructions
- add some missing configuration variables
- remove problematic html install --set instructions
- fix NOTES.TXT for serviceType of NodePort, assuming minikube

### CODE REVIEW

**Comment by bryanlarsen** [comment]
CLA signed.

**Comment by seanknox** [comment]
@k8s-bot ok to test

File: stable/magento/README.md [hunk 1]
```diff
@@ -59,18 +59,31 @@ The following tables lists the configurable parameters of the Magento chart and
...
 The above parameters map to the env variables defined in [bitnami/magento](http://github.com/bitnami/bitnami-docker-magento). For more information please refer to the [bitnami/magento](http://github.com/bitnami/bitnami-docker-magento) image documentation.
 
+A YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+$ helm inspect values stable/magento > values.yaml
+$ vi values.yaml
+$ helm install --name my-release -f values.yaml stable/magento
+
+### Installing on GKE
```
Comment (sameersbn) [inline stable/magento/README.md]: You have a typo here, the code block should be closed before the `Installing on GKE` section start.

File: stable/magento/README.md [hunk 2]
```diff
@@ -59,18 +59,31 @@ The following tables lists the configurable parameters of the Magento chart and
...
+
+```
 > **Note**:
```
Comment (sameersbn) [inline stable/magento/README.md]: I see you have moved this into a Installing on GKE section. But the whole section looks missing/incomplete.

File: stable/magento/README.md [hunk 3]
```diff
@@ -85,23 +98,69 @@ The above parameters map to the env variables defined in [bitnami/magento](http:
...
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+### Installing on minikube
 
-```console
-$ helm install --name my-release \
-  --set magentoUsername=admin,magentoPassword=password,mariadb.mariadbRootPassword=secretpassword \
-    stable/magento
-```
+    $ helm inspect values stable/magento > values.yaml
```
Comment (sameersbn) [inline stable/magento/README.md:103]: You should use command formatting syntax here, i.e.

`$ helm inspect values stable/magento > values.yaml`

File: stable/magento/README.md [hunk 4]
```diff
@@ -85,23 +98,69 @@ The above parameters map to the env variables defined in [bitnami/magento](http:
...
+- magentoHost
+- magentoPassword  (requires both letters and numbers)
+- magentoEmail
+- mariadb.mariadbRootPassword
+- mariadb.persistence.storageClass
+- serviceType: NodePort
+- persistence.apache.storageClass
+- persistence.magento.storageClass
+
+    $ helm install --name my-release -f values.yaml stable/magento
```
Comment (sameersbn) [inline stable/magento/README.md:141]: Use command formatting syntax here as well.

**Review by sameersbn — CHANGES_REQUESTED** [review]
I see you have added section on installing on GKE, Minikube and Metal. In my opinion you only need to specify the right `serviceType`. All other parameters will be the same isn't it?

File: stable/magento/README.md (hunk 2 above)
Comment (bryanlarsen) [reply stable/magento/README.md]: Yup, that's existing text.   I don't have a GKE account.

**Comment by bryanlarsen** [comment]
"All other parameters will be the same isn't it?"

No, requirements and persistence are different.

**Comment by viglesiasce** [comment]
@bryanlarsen can you update as per @sameersbn's requests?

@sameersbn can you just make the changes and merge it in?

**Comment by bryanlarsen** [comment]
I'm not using this chart any longer, so go ahead and update as you see fit.

**Comment by seanknox** [comment]
Closing as it's stale. Feel free to re-open if you like.

**Pull request closed without merging**